test/email-field-events.html

```diff
@@ -72,6 +78,39 @@
           });
         });
```

Comment:
It seems the tests can be reorganized a bit to avoid having to introduce a separate fixture. It's possible because for these tests it's not crucial whether the clear button is enabled via the attribute or property.


```js
describe('has-input-value-changed-event', () => {
  beforeEach(async() => {
    emailField = fixture('email-field');
    input = emailField.inputElement;
    hasInputValueChangedSpy = sinon.spy();
    valueChangedSpy = sinon.spy();
    emailField.addEventListener('has-input-value-changed', hasInputValueChangedSpy);
    emailField.addEventListener('value-changed', valueChangedSpy);
  });

  ...

  describe('with clear button', () => {
    beforeEach(() => {
      emailField.clearButtonVisible = true;
    });

    describe('with user input', () => {
      ...
    });
  });
});
```